public/index.html

```diff
@@ -19,13 +19,20 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Newsboard</title>
   </head>
-  <body>
+  <body style="margin: 0">
```

Comment:
Ahh! Yes.  That one. I wrote it inline to avoid creating another styles.css file just for this one rule.